Library/Homebrew/test/description_cache_store_spec.rb

```diff
@@ -33,6 +33,15 @@
       expect(database).to receive(:empty?).at_least(:once).and_return(false)
       cache_store.update_from_report!(report)
     end
+
+    context "without HOMEBREW_EVAL_ALL" do
+      before { allow(Homebrew::EnvConfig).to receive(:eval_all?).and_return(false) }
+
+      it "clears the database" do
+        expect(database).to receive(:clear)
```

Comment:
This duplicates the tests above so I'm even less convinced it's needed.

Reply:
I thought this was actually the more useful test, as it covers the branch I added. Removed for now, but I do have it backed up on another branch in my fork if it's needed.